.circleci/config.yml

```diff
@@ -36,23 +36,28 @@ job-references:
       - checkout
       - run: *setup_environment
       - run: *install_dependencies
+      - run:
+          name: "composer"
+          command: |
+            sudo mkdir ../../plugins
+            sudo chown circleci:circleci ../../plugins
+            composer install --dev
+            composer require "phpunit/phpunit=5.7.*"
       - run:
           name: "phpcs"
           command: |
             mkdir -p /tmp/test-results/phpcs
-            composer global require wp-coding-standards/wpcs
-            phpcs --config-set installed_paths $HOME/.composer/vendor/wp-coding-standards/wpcs
-            phpcs --report=junit --report-file=/tmp/test-results/phpcs/$CIRCLE_STAGE.xml
+            vendor/bin/phpcs --config-set installed_paths vendor/wp-coding-standards/wpcs
+            vendor/bin/phpcs --report=junit --report-file=/tmp/test-results/phpcs/$CIRCLE_STAGE.xml
       - run:
           name: "phpunit"
           when: always
           command: |
             mkdir -p /tmp/test-results/phpunit /tmp/test-results/phpunit-multisite
-            composer global require "phpunit/phpunit=5.7.*"
             rm -rf $WP_TESTS_DIR $WP_CORE_DIR
             bash bin/install-wp-tests.sh wordpress_test root '' 127.0.0.1 latest
-            phpunit --log-junit /tmp/test-results/phpunit/$CIRCLE_STAGE.xml
-            WP_MULTISITE=1 phpunit --log-junit /tmp/test-results/phpunit-multisite/$CIRCLE_STAGE.xml
+            vendor/bin/phpunit --log-junit /tmp/test-results/phpunit/$CIRCLE_STAGE.xml
+            WP_MULTISITE=1 vendor/bin/phpunit --log-junit /tmp/test-results/phpunit-multisite/$CIRCLE_STAGE.xml
```

Comment:
I think yes we could remove WP_MULTISITE testing for now. Perhaps another PR after this one builds successfully.